packages/treat/src/webpack-plugin/loader.js

```diff
@@ -233,24 +236,15 @@ async function produce(loader) {
     cssRequests.forEach((request) => ownedCssRequests.add(request));
   }
 
-  return serializeTreatModule(
+  const r = serializeTreatModule(
```

Comment:
Does this still benefit from being declared rather than returned?
```suggestion
  return serializeTreatModule(
```